src/plugins/vue/VuePlugin.ts

```diff
@@ -94,9 +94,9 @@ export class VueComponentClass implements Plugin {
 
   public bundleEnd(context: WorkFlowContext) {
     if (context.useCache && this.hasProcessedVueFile) {
-      const haveGlobal = context.isGlobalyIgnored('process')
-      const process = haveGlobal ? 'global.process' : 'process'
-      const importProcess = haveGlobal ? '' : 'var process = FuseBox.import(\'process\');\n\n'
+      const hasGlobal = context.isGlobalyIgnored('process')
+      const process = hasGlobal ? 'global.process' : 'process'
+      const importProcess = hasGlobal ? '' : 'var process = FuseBox.import(\'process\');\n\n'
```

Comment:
Just noticed why do we need to define process globally?

Reply:
I use global process from global because the simple variable process is not defined, this is strange because when i test with `this` context that is work.. I don't know the reason of the bug.

An probably solution it is add call the anonymous function with process argument.